tools/lang/src/cli.rs

```diff
@@ -119,17 +119,19 @@ impl Cli {
                 let ast = parse_str_to_ast(&contents)?;
                 let programs = compile_ast_to_restrictions(ast)?;
 
-                println!("Successfully parsed the following programs:");
-                if *verbose {
-                    for program in &programs {
-                        println!("{}", String::from_utf8(program.name.to_vec()).unwrap());
-                        let program_str = serde_json::to_string(program).unwrap();
-                        println!("JSON: {}", program_str);
+                println!("Successfully compiled the following programs:");
```

Comment:
What do you think : ~~Successfully compiled the following programs~~

Successfully compiled the following program(s)

Reply:
Probably better but I want this merged and I have approvals. Will make the change locally and include it in whatever the next PR is to this repo